phoenix-core/src/it/java/org/apache/phoenix/rpc/UpdateCacheConnectionLevelPropIT.java

```diff
@@ -61,6 +62,9 @@ public class UpdateCacheConnectionLevelPropIT extends ParallelStatsDisabledIT {
     private static Connection conn1;
     private static Connection conn2;
     private static ConnectionQueryServices spyForConn2;
+    private boolean isLastDDLTimestampValidationEnabled
```

Comment:
Why not to get the last ddl timestamp validation enabled flag from the connection properties?

Reply:
Not sure if there is a connection here in the test to use?